examples/components/http-server-hono/src/component.ts

```diff
@@ -0,0 +1,3 @@
+import { app } from "./app.js";
+
+app.fire();
```

Comment:
This is the magic sauce, think it's worth explaining

```suggestion
// Register hono with the global fetch listener as supported by the underlying StarlingMonkey JS runtime.
// See: https://github.com/bytecodealliance/ComponentizeJS?tab=readme-ov-file#using-starlingmonkeys-fetch-event
app.fire();
```

Reply:
Thanks for this, it's certainly worth explaining -- this is actually covered in the README, but explicit coverage near code is likely to hit people who won't read the README.